src/scripts/ci_check_install.py

```diff
@@ -75,6 +76,34 @@ def verify_includes(build_config):
 
     return True
 
+def verify_cmake_package(build_config):
+    if build_config['os'] not in ['windows', 'linux', 'macos']:
+        return True # skip (e.g. for mingw)
+
+    cmake_build_dir = os.path.join(build_config['abs_root_dir'], build_config['build_dir'], 'cmake_test')
+    cmake_test_dir = os.path.join(build_config['abs_root_dir'], "src", "tests", "cmake")
+
+    def cmake_preset():
+        if build_config['os'] == 'windows':
+            return 'windows_x86_64' if build_config['arch'] == 'x86_64' else 'windows_x86'
+        return 'unix'
+
+    def test_target():
+        return 'test' if build_config['os'] != 'windows' else 'RUN_TESTS'
+
+    try:
+        subprocess.run(["cmake", "--preset", cmake_preset(),
+                                 "-B", cmake_build_dir,
+                                 "-S", cmake_test_dir,
+                                 "-DCMAKE_PREFIX_PATH=%s" % build_config['prefix']], check=True)
+        subprocess.run(["cmake", "--build", cmake_build_dir, "--config", "Release"], check=True)
+        subprocess.run(["cmake", "--build", cmake_build_dir, "--config", "Release", "--target", test_target()], check=True)
```

Comment:
This basically exercises the creation of a CMake project that consumes Botan via the `-config.cmake` files. It is built on the CI (in `static`, `shared`, `nist` and `bsi` configs on Windows, Linux and macOS).

Reply:
Seems fine to me to test it this way.